File: sdk/go/entity/recon_order_refund_split_sub.go
package entity

import (
	"time"

	"github.com/shopspring/decimal"
)

// ReconOrderRefundSplitSub 对账订单退款分账子记录
type ReconOrderRefundSplitSub struct {
	ID                int64           `gorm:"primaryKey;autoIncrement;comment:主键 ID"`
	OrderNo           string          `gorm:"size:64;not null;comment:订单号"`
	SubOrderNo        string          `gorm:"size:64;comment:子订单号"`
	MerchantId        string          `gorm:"size:64;not null;comment:商户 ID"`
	MerchantOrderNo   string          `gorm:"size:64;comment:商户原始订单号"`
	RefundSplitAmount decimal.Decimal `gorm:"type:decimal(18,2);not null;comment:退款分账金额"`
	CreateTime        time.Time       `gorm:"autoCreateTime;comment:创建时间"`
	UpdateTime        time.Time       `gorm:"autoUpdateTime;comment:更新时间"`
}

File: sdk/python/entity/recon_order_refund_split_sub.py
from dataclasses import dataclass
from datetime import datetime
from typing import Optional

@dataclass
class ReconOrderRefundSplitSub:
    """退款分账子记录"""
    order_no: Optional[str] = None
    sub_order_no: Optional[str] = None
    merchant_id: Optional[str] = None
    merchant_order_no: Optional[str] = None
    refund_split_amount: Optional[float] = None
    
    create_time: Optional[datetime] = None
    update_time: Optional[datetime] = None
